config/os.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+import "fmt"
+
+const (
+	WindowsOS = "windows"
+	LinuxOS   = "linux"
+)
+
+func ValidateOS(os string) error {
```

Comment:
This is Docker OS? ImageOS? or host machine OS? It's a bit unclear from the file alone